java/org/contikios/cooja/contikimote/interfaces/ContikiEEPROM.java

```diff
@@ -80,20 +79,13 @@
  * @author Claes Jakobsson (based on ContikiCFS by Fredrik Osterlind)
  */
 @ClassDescription("EEPROM")
-public class ContikiEEPROM extends Observable implements MoteInterface, PolledAfterActiveTicks {
+public class ContikiEEPROM implements MoteInterface, PolledAfterActiveTicks {
   private static final Logger logger = LogManager.getLogger(ContikiEEPROM.class);
 
   public static final int EEPROM_SIZE = 1024; /* Configure EEPROM size here and in eeprom.c. Should really be multiple of 16 */
   private final Mote mote;
   private final VarMemory moteMem;
-
-  private final JTextArea dataViewArea;
-  private final JLabel lastTimeLabel;
-  private final JLabel lastReadLabel;
-  private final JLabel lastWrittenLabel;
-
-  private int lastRead = 0;
-  private int lastWritten = 0;
```

Comment:
I would suggest keeping these counters as instance variables so they are not reset when no visualizer is open.

Reply:
Restored.